pkg/host-disk/host-disk.go

```diff
@@ -80,22 +83,29 @@ func ReplacePVCByHostDisk(vmi *v1.VirtualMachineInstance, clientset kubecli.Kube
 				continue
 			}
 
-			pvc, exists, isBlockVolumePVC, err := types.IsPVCBlockFromClient(clientset, vmi.Namespace, volumeSource.PersistentVolumeClaim.ClaimName)
-			if err != nil {
-				return err
-			} else if !exists {
-				return fmt.Errorf("persistentvolumeclaim %v not found", volumeSource.PersistentVolumeClaim.ClaimName)
-			} else if isBlockVolumePVC {
+			volumeStatus, ok := pvcVolume[volume.Name]
```

Comment:
do you mean instead of ensuring that the volume status exists and that the PersistentVolumeClaimInfo pointer is set and that volume mode isnt' block... that we'd simply call something like the following?

`if pvcVolume[volume.Name].PersistentVolumeClaimInfo.VolumeMode == k8sv1.PersistentVolumeBlock {
    continue
}
`

probably... but if we're ever wrong about this assumption then the process crashes due to trying to dereference a null pointer.

Reply:
Fair. If we are ever going to end up in this situation(without `VolumeMode` info) shouldn't we rather return an error?